scrapy/spidermiddlewares/urllength.py

```diff
@@ -14,22 +14,27 @@
 
 class UrlLengthMiddleware:
 
-    def __init__(self, maxlength):
+    def __init__(self, maxlength, stats):
```

Comment:
Can we make the parameter optional, to maintain backward compatibility? (think of class inheritance and calls to `super().__init__(maxlength)`).